ra2ce/analysis/direct/damage/max_damage.py

```diff
@@ -45,10 +45,11 @@ def __init__(self, name=None, damage_unit=None):
         self.name = name
         self.damage_unit = damage_unit
 
-    def from_csv(self, path: Path, sep=",") -> None:
+    def from_csv(self, path: Path, sep=",", output_unit='euro/m') -> None:
```

Comment:
when is this ever in our code being used?  Please remove it and set it as a variable within the code
```suggestion
    def from_csv(self, path: Path, sep=",") -> None:
      default_output_unit="euro/m"
```